packages/frontend/src/app/components/login/MerchantConnectWallet.tsx

```diff
@@ -113,6 +113,7 @@ const MerchantConnectWallet = ({ close }: { close: () => void }) => {
             clientWallet,
             false,
             shopId,
+            new URL(window.location.href),
```

Comment:
@wanderer ended up doing this to fix the phishing warning. Not sure if this is the best way of going about it but it works.